hack/build-images.sh

```diff
@@ -138,6 +138,9 @@ image openshift/origin-sti-builder           images/builder/docker/sti-builder
 image openshift/origin-f5-router             images/router/f5
 image openshift/node                         images/node
 
+# images used for tests
+image openshift/test-multicast               images/testing/multicast
```

Comment:
Set up a repo in the openshift org that has a dockerfile and set an automated build via docker hub is how we normally address this.